fix TestISequencer_Start

https://github.com/voedger/voedger/actions/runs/14485203022
```
  --- FAIL: TestLongRace (0.03s)
      isequencer_test.go:207: 
          	Error Trace:	/home/runner/work/voedger/voedger/pkg/isequencer/isequencer_test.go:207
          	Error:      	Should be false
          	Test:       	TestLongRace
      --- FAIL: TestLongRace/should_reject_when_actualization_in_progress (0.00s)
```

--- a/pkg/isequencer/isequencer_test.go
+++ b/pkg/isequencer/isequencer_test.go
@@ -194,11 +194,19 @@ func TestISequencer_Start(t *testing.T) {
 	})
 
 	t.Run("should reject when actualization in progress", func(t *testing.T) {
+		storage := createDefaultStorage()
+		stuckActualizationCh := make(chan any)
 		seq, cleanup := isequencer.New(params, storage, iTime)
 		defer cleanup()
 
 		offset := isequencer.WaitForStart(t, seq, 1, 1, true)
-		require.Equal(isequencer.PLogOffset(2), offset)
+		require.Equal(isequencer.PLogOffset(1), offset)
+
+		// force actualization to stuck to gaurantee that Start will be rejected
+		storage.SetOnActualizeFromPLog(func() {
+			<-stuckActualizationCh
+		})
+		defer close(stuckActualizationCh)
 
 		seq.Actualize()
 

--- a/pkg/isequencer/isequencer_test.go
+++ b/pkg/isequencer/isequencer_test.go
@@ -202,7 +202,7 @@ func TestISequencer_Start(t *testing.T) {
 		offset := isequencer.WaitForStart(t, seq, 1, 1, true)
 		require.Equal(isequencer.PLogOffset(1), offset)
 
-		// force actualization to stuck to gaurantee that Start will be rejected
+		// force actualization to stuck to guarantee that Start will be rejected
 		storage.SetOnActualizeFromPLog(func() {
 			<-stuckActualizationCh
 		})

